misc.go

```diff
@@ -327,6 +331,7 @@ var tokenMap = map[string]int{
 	"IS":                       is,
 	"ISSUER":                   issuer,
 	"ISOLATION":                isolation,
+	"USE_TOJA":                 hintUseToja,
```

Comment:
What is `toja`?

Reply:
Convert `in subquery` into inner join and aggregation. Seems like this one.
The name is advised by product team.

Reply:
How about adding a comment for this abbreviation.

Reply:
Okay, I will contact PM team also.

Reply:
Okay, and I will contact PM team also.